bookkeeper-server/src/main/java/org/apache/bookkeeper/client/PendingAddOp.java

```diff
@@ -256,6 +258,12 @@ public void writeComplete(int rc, long ledgerId, long entryId, BookieSocketAddre
         }
 
         if (completed) {
+            if (rc != BKException.Code.OK) {
+                // Got an error after satisfying AQ. This means we are under replicated at the create itself.
+                // Update the stat to reflect it.
+                long latencyNanos = MathUtils.elapsedNanos(requestTimeNanos);
+                addOpUrLogger.registerFailedEvent(latencyNanos, TimeUnit.NANOSECONDS);
```

Comment:
do we need `OpStatsLogger` here or just a simple counter is good enough? 

or in a different way, is the latency metric here helpful?

Reply:
Initially, I was thinking of getting latency as we will know how long the failure took. But may be I can simply swithch to a counter.